src/stats/probs/cdf.rs

```diff
@@ -226,6 +239,8 @@ impl<T: Ord> CDF<T> {
     }
 
     /// Provide an iterator over the PMF corresponding to this CDF.
+    ///
+    /// Runtime complexity: O(n), where n is the number of `Entry`s in `CDF`.
```

Comment:
```suggestion
```
I'd not state runtime complexity here, since technically this function's runtime is O(1), as it only builds an iterator, which doesn't perform any computations until it is consumed.

Reply:
Thanks for that insight! I was not sure about that, so I'm glad you did have a thorough look! :D